ark/segmentation/marker_quantification.py

```diff
@@ -5,16 +5,133 @@
 
 import xarray as xr
 
-from skimage.measure import regionprops_table
+from skimage.measure import regionprops, regionprops_table
 
 from ark.utils import io_utils, load_utils, misc_utils, segmentation_utils
-from ark.segmentation.signal_extraction import extraction_function
+from ark.segmentation.signal_extraction import EXTRACTION_FUNCTION
+from ark.segmentation.regionprops_extraction import REGIONPROPS_FUNCTION
 
 import ark.settings as settings
 
 
+def compute_extra_prop_info(prop_info, regionprops_extras, **kwargs):
+    """Derives new features specified by regionprops_extras from a regionprops features
+
+    Args:
+        prop_info (skimage.measure.regionprops):
+            A list of property information returned by regionprops
+        regionprops_extras (list):
+            A list of regionprops features to compute, each value should correspond to
+            a function in the global regionprops_functions variable
+        **kwargs:
+            Arbitrary keyword arguments
+
+    Returns:
+        pandas.DataFrame:
+            A dataframe with columns corresponding to each regionprops_extras feature
+    """
+
+    misc_utils.verify_in_list(
+        regionprops_extras=regionprops_extras,
+        regionprops_extras_options=list(REGIONPROPS_FUNCTION.keys())
+    )
+
+    # define an empty list for each regionprop feature
+    prop_extra_data = {re: [] for re in regionprops_extras}
+
+    # generate the required data for each cell
+    for prop in prop_info:
+        for re in regionprops_extras:
+            prop_extra_data[re].append(REGIONPROPS_FUNCTION[re](prop, **kwargs))
+
+    # convert the dictionary to a DataFrame
+    prop_extra_df = pd.DataFrame.from_dict(prop_extra_data)
+
+    return prop_extra_df
+
+
+def get_cell_props(segmentation_labels, regionprops_features, regionprops_extras=None, **kwargs):
+    """Gets regionprops features from the provided segmentation labels
+
+    Args:
+        segmentation_labels (numpy.ndarray):
```

Comment:
Specify that this represents segmentation from a single compartment from a single FOV